artcommon/artcommonlib/util.py

```diff
@@ -107,6 +107,7 @@ def get_assembly_release_date(assembly, group):
     """
     Get assembly release release date from release schedule API
     """
+    assembly_release_data = None
```

Comment:
typo alert :lady_beetle: 
```suggestion
    assembly_release_date = None
```